web/src/Cell.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { Component, ChangeEvent, FormEvent } from 'react'
+import { Form } from 'semantic-ui-react'
```

Comment:
@NovikovAntonY after a phone talk we decided to drop `semantic-ui-react` from this project. So, my suggestion is to replace any `semantic-ui-react` component with its html-plain corresponding tag.

Reply:
@rianby64 ok, I will. Now i preparing for commit delete row feature. After this begin to remove semantic UI from project.

Reply:
@rianby64 done